docs/src/guide/deployment/theming.md

```
@@ -0,0 +1,307 @@
+volumes:
+  - name: authup-theme
+    configMap:
+      name: authup-theme
+```
+
+Every file below is text, so one ConfigMap holds the lot. Binary assets go in
+its `binaryData`.
+
+## Layout
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)